src/lib/transaction_logic/test/transaction_logic.ml

```diff
@@ -5,11 +5,10 @@ open Mina_numbers
 open Signature_lib
 open Helpers
 open Protocol_config_examples
+open Unsigned
 open Zkapp_cmd_builder
 
-(* The function under test is quite slow, so keep the trials count low
```

Comment:
Is this comment no longer true? It feels like we should stick with the lower number if this has a significant impact on CI time.

Reply:
I think this comment is no longer true. Let me check, how much exactly do these tests take to execute.

Reply:
This whole test suite comiles and runs in under 8 seconds on my crappy laptop (provided everything else is already built). So I think we're fine.